orc8r/gateway/python/scripts/show_gateway_info.py

```diff
@@ -14,12 +14,25 @@
 """
 
 import argparse
+import os
 import textwrap
 
+
 import snowflake
 from magma.common.cert_utils import load_public_key_to_base64der
 
 
+def get_commit_info() -> str:
+    """"get commit info if available"""
+    return "Commit Branch: {}\n" \
+            "Commit Tag: {}\n" \
+            "Commit Hash: {}\n" \
+            "Commit Date: {}".format(os.getenv("MAGMA_BUILD_BRANCH"),
+                                    os.getenv("MAGMA_BUILD_TAG"),
+                                    os.getenv("MAGMA_BUILD_COMMIT_HASH") or os.getenv("COMMIT_HASH"),
```

Comment:
you may want to use os.getenv() with defaults: os.getenv("MAGMA_BUILD_BRANCH", "UNKNOWN") for better readability if the vars are not set

Reply:
sure, but note per the image they will always be set (either to a value or to unknown)